lib/reporting/proofing_rate_report.rb

```diff
@@ -8,20 +8,22 @@ module Reporting
   class ProofingRateReport
     DATE_INTERVALS = [30, 60, 90].freeze
 
-    attr_reader :end_date, :wait_duration
+    attr_reader :end_date, :wait_duration, :by_month
 
     def initialize(
       end_date:,
       verbose: false,
       progress: false,
       wait_duration: CloudwatchClient::DEFAULT_WAIT_DURATION,
-      parallel: false
+      parallel: false,
+      by_month: false
     )
       @end_date = end_date.in_time_zone('UTC')
       @verbose = verbose
       @progress = progress
       @wait_duration = wait_duration
-      @parallel = parallel
+      @parallel = false # remove me, 50/50 state risk?
```

Comment:
not a 50/50 state risk since `ProofingRateReport` is not itself a background job and its arguments are never serialized